src/main/java/spoon/support/compiler/SpoonPom.java

```diff
@@ -503,7 +503,7 @@ private void generateClassPathFile(File mvnHome, MavenLauncher.SOURCE_TYPE sourc
 					sb.append(line);
 					line = br.readLine();
 				}
-				if (!"".equals(sb.toString())) {
+				if (!sb.toString().isEmpty()) {
```

Comment:
This can be further simplified to `!sb.isEmpty()`

Reply:
Good catch time for the next refactoring

Reply:
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/CharSequence.html#isEmpty() This is a JDK15 method